crates/libcontainer/src/workload/mod.rs

```diff
@@ -1,4 +1,5 @@
 use oci_spec::runtime::Spec;
+use std::{collections::HashMap, env};
```

Comment:
Formatting issue:
```suggestion
use std::collections::HashMap;
use std::env;
```

You can catch formatting issues locally by running `just lint`.

Reply:
fixed in 6471f71